src/int/add.rs

```diff
@@ -0,0 +1,206 @@
+//! [`Int`] addition operations.
+
+use core::ops::{Add, AddAssign};
+
+use num_traits::WrappingAdd;
+use subtle::{ConstantTimeEq, CtOption};
+
+use crate::{Checked, CheckedAdd, Int, Wrapping};
+
+impl<const LIMBS: usize> Int<LIMBS> {
+    /// Perform checked addition.
+    fn checked_add_internal(&self, rhs: &Self) -> CtOption<Self> {
```

Comment:
nit: remove _internal prefix to be consistent with Uint

Reply:
I did this because, indeed, it makes things consistent. However (!), there are now two `checked_add`s for `Int`:
- The `ConstCtOption`, and
- The `CtOption` (trait) version.

Reply:
The same thing is for Uint there are two versions of the same method (one from trait impl one from struct impl) take bitand as an example:
https://github.com/RustCrypto/crypto-bigint/blob/master/src/uint/bit_and.rs#L11
https://github.com/RustCrypto/crypto-bigint/blob/master/src/uint/bit_and.rs#L55

The only difference is that one is const fn, the other is not (I believe due to lack of stability of this feature yet).

Reply:
done!